package.json

```diff
@@ -13,10 +13,20 @@
     "chai": "^4.2.0"
   },
   "devDependencies": {
-    "cypress": "^4.5.0"
+    "cypress": "^4.5.0",
+    "cypress-multi-reporters": "^1.4.0",
+    "mocha": "^8.0.1",
+    "mochawesome": "^6.1.1",
+    "mochawesome-merge": "^4.1.0",
+    "mochawesome-report-generator": "^5.1.0"
   },
     "scripts": {
-      "chrome:headless": "cypress run --headless --browser chrome",
-      "chrome:head": "cypress run --browser chrome"
+      "clean:reports": "rm -R -f cypress/reports && mkdir cypress/reports",
```

Comment:
2-nd part of the command `mkdir cypress/reports` is not necessary, because `reports` node will be created by mochawesome reporter by default. Please remove it.

Reply:
Could you clarify, please?
Should we leave `"clean:reports": "rm -R -f cypress/reports"` as is or combine with `"cypress:report"`?

Reply:
My current comment is for updating `"clean:reports"` command.
Just make "clean:reports": `"rm -R -f cypress/reports"`

Reply:
My [second comment](https://github.com/volhadl/aqajs/pull/11#discussion_r452691014) is to create 1 command instead of separate 2 commands:
"cypress:tests": "rm -R -f cypress/reports && cypress run"